packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -25,20 +25,72 @@ import { IDetailedHbarSpendingPlan } from '../../db/types/hbarLimiter/hbarSpendi
 import { SubscriptionType } from '../../db/types/hbarLimiter/subscriptionType';
 import { Logger } from 'pino';
 import { formatRequestIdMessage } from '../../../formatters';
+import { Counter, Gauge, Registry } from 'prom-client';
```

Comment:
Nit: Warning `Import "Registry" is only used as types`. 

Perhaps add the following to `eslintrc.js`?
` "@typescript-eslint/consistent-type-imports": "off"`

Reply:
Done, fixed the eslint rules